### PR TITLE
Add metrics to the ade-corpus-v2 dataset

### PR DESCRIPTION
**Pull request by tae898**
I used the below three rules to add metrics:

1. If specific words have to be in the answer, the metric is accuracy.
2. If a text has to be generated in the answer, the metrics are BLEU and ROUGE.

~~3. If both 1 and 2 have to be satisfied, the metrics are accuracy, BLEU, and ROUGE.~~

### CODE REVIEW

File: promptsource/templates/ade_corpus_v2/Ade_corpus_v2_classification/templates.yaml [hunk 1]
```diff
@@ -2,28 +2,28 @@ dataset: ade_corpus_v2
...
+      metrics:
+      - Accuracy
       original_task: true
-    name: baseline
+    name: binary-classification
     reference: ''
   78c4ce65-dd66-46ed-878d-11f4eca5e544: !Template
-    answer_choices: Yes, it is related to adverse drug effect. ||| No, it is not related
-      to adverse drug effect.
+    answer_choices: Yes ||| No
```
Comment (stephenbach) [inline promptsource/templates/ade_corpus_v2/Ade_corpus_v2_classification/templates.yaml:16]: This doesn't follow the required format of responding A or B if those are used to indicate choices. More generally, these just seem like weird ways of phrasing the question. Maybe both the classification tasks should just use Yes and No as the targets.

**Review by stephenbach — CHANGES_REQUESTED** [review]
Thanks! I think this dataset still needs some work.

- For the classification subset, I left a note about a minor formatting thing.
- For the other two, I think more work is needed. A lot of the prompts don't use the contextual information, resulting in prompts like
```Input

What drug has a dosage of 1500 mg/m2?
Target

methotrexate
```
This doesn't really make sense, because there's no context from which to answer the question. I think making these more like `squad` would be a lot better.

**Comment by tae898** [comment]
Thank you for the nice feedback again. I've made some changes. Please let me know if further changes are necessary.

**Comment by stephenbach** [comment]
Thanks! These look great now! Just wondering, why are no prompts marked original for the relation subsets?

**Comment by tae898** [comment]
> Thanks! These look great now! Just wondering, why are no prompts marked original for the relation subsets?

AFAIK, the original authors created this dataset to train a machine-learning based classifier. The annotations from the relation subsets were used to compare the annotations between the human annotators, but they were not used to train an automatic classifier.

**Comment by stephenbach** [comment]
Ah, gotcha. I think it would be fine to mark them as original task, because it's still the intended task, even if they originally used it more for testing humans.

**Comment by tae898** [comment]
> Ah, gotcha. I think it would be fine to mark them as original task, because it's still the intended task, even if they originally used it more for testing humans.

alright!

**Review by stephenbach — APPROVED** [review]
Thanks for all your hard work on this PR! Merging now!

**Pull request merged**